src/hiscores.ts

```diff
@@ -95,9 +102,9 @@ export async function getRSNFormat(
     if (anchor) {
       return rsnFromElement(anchor);
     }
-    throw Error(PLAYER_NOT_FOUND_ERROR);
-  } catch {
-    throw Error(PLAYER_NOT_FOUND_ERROR);
+    throw new PlayerNotFoundError();
+  } catch (err) {
```

Comment:
Can remove this argument if we aren't passing it to the `HiScoresError` constructor.